pkgs/firedragon/firedragon.nix

```
@@ -58,7 +58,7 @@ rec {
   inherit packageVersion;
 
   src = fetchurl {
-    url = "https://gitlab.com/api/v4/projects/55893651/packages/generic/firedragon/${packageVersion}/firedragon-v${packageVersion}.source.tar.zst";
+    url = "https://gitlab.com/api/v4/projects/69949446/packages/generic/firedragon/v${packageVersion}/firedragon-source-v${packageVersion}.tar.zst";
```

I suggest to go through the release to fetch the source archive:

```suggestion
    url = "https://gitlab.com/garuda-linux/firedragon/firedragon12/-/releases/v${packageVersion}/downloads/firedragon-source.tar.zst";
```

---

IIRC that archive is very different and much smaller. Also was just trying to follow the existing pattern.

---

Nevermind, they redirect to the same. I just meant that the git tree as-is isn't the same as what you get from CI, but that makes sense.